apps/site/src/app/(home)/sections/FAQ/FAQ.module.scss

```diff
@@ -1,121 +1,125 @@
 @use "zothacks-theme" as theme;
 @use "bootstrap-utils" as utils;
 
+$item-padding-y: 32px;
+$item-padding-x: 48px;
+
+$answer-font-size: 16px;
+$question-font-size: 20px;
+
+.light-blue-text {
+	color: theme.$light-blue;
+}
+
+.purple-text {
+	color: theme.$purple;
+}
+
 .faq {
 	position: relative;
 	display: flex;
 	flex-direction: column;
-	width: 80%;
+	width: 90%;
```

Comment:
Suggestion: rather than manually constraining the inner elements, we can use Bootstrap's native container sizing (unfortunately not done last year) by changing the main `<section>` element to be a Container, see Home Intro ([Intro.tsx:14](https://github.com/HackAtUCI/zothacks-site/blob/main/apps/site/src/app/(home)/sections/Intro/Intro.tsx#L14)) for example.